ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/basefee-input.test.js

```diff
@@ -53,38 +55,26 @@ const render = (txProps) => {
 };
 
```

Comment:
Test cases will also require change once this is fixed: https://github.com/MetaMask/metamask-extension/pull/13099/files#r770360839